### PR TITLE
Do not use CmdRunner.Capture() while executing a command with sudo

### LINKED ISSUE #1440: Yay hangs after parsing SRCINFO
### Affected Version

yay v10.1.2.r0.g7d849a8 - libalpm v12.0.2

### Describe the bug

After parsing the SRCINFO, Yay installs the requirements, but hangs after Pacman installs the packages. I need to ^C then restart the process to install the program. After ^C, the package is installed, so the dependency hang doesn't happen. 

### Reproduction Steps
1. Install a package with yay that doesn't have dependencies fulfilled
2. Yay gets stuck after dependency package(s) is/are installed
3. ^C, rerun yay command to finish install

### Expected behavior

I expected the installation of the package to continue as normal, and not hang.

### Output

```sh
{
	"aururl": "https://aur.archlinux.org",
	"buildDir": "/home/username/.cache/yay",
	"absdir": "/home/username/.cache/yay/abs",
	"editor": "",
	"editorflags": "",
	"makepkgbin": "makepkg",
	"makepkgconf": "",
	"pacmanbin": "pacman",
	"pacmanconf": "/etc/pacman.conf",
	"redownload": "no",
	"rebuild": "no",
	"answerclean": "",
	"answerdiff": "",
	"answeredit": "",
	"answerupgrade": "",
	"gitbin": "git",
	"gpgbin": "gpg",
	"gpgflags": "",
	"mflags": "",
	"sortby": "votes",
	"searchby": "name-desc",
	"gitflags": "",
	"removemake": "ask",
	"sudobin": "sudo",
	"sudoflags": "",
	"requestsplitn": 150,
	"sortmode": 0,
	"completionrefreshtime": 7,
	"sudoloop": false,
	"timeupdate": false,
	"devel": false,
	"cleanAfter": false,
	"provides": true,
	"pgpfetch": true,
	"upgrademenu": true,
	"cleanmenu": true,
	"diffmenu": true,
	"editmenu": false,
	"combinedupgrade": false,
	"useask": false,
	"batchinstall": false
}

```

### PR DESCRIPTION
**Pull request by benjarobin**
Fix #1440

Do not hesitate to do a code review, since I really do not know Go...

### CODE REVIEW

File: install.go [hunk 1]
```diff
@@ -29,33 +29,33 @@ import (
...
 
 	cmdArgs = cmdArgs.CopyGlobal()
-	_ = cmdArgs.AddArg("D", "asdeps")
+	_ = cmdArgs.AddArg("q", "D", "asdeps")
 	cmdArgs.AddTarget(pkgs...)
-	_, stderr, err := config.Runtime.CmdRunner.Capture(passToPacman(cmdArgs), 0)
+	err = config.Runtime.CmdRunner.Show(passToPacman(cmdArgs))
 	if err != nil {
-		return fmt.Errorf("%s %s", stderr, err)
+		return fmt.Errorf(gotext.Get("error updating package install reason to non-explicitly"))
```
Comment (Jguer) [inline install.go]: I'd change to `error updating package install reason to dependency`

File: install.go [hunk 2]
```diff
@@ -29,33 +29,33 @@ import (
...
 
 	cmdArgs = cmdArgs.CopyGlobal()
-	_ = cmdArgs.AddArg("D", "asexplicit")
+	_ = cmdArgs.AddArg("q", "D", "asexplicit")
 	cmdArgs.AddTarget(pkgs...)
-	_, stderr, err := config.Runtime.CmdRunner.Capture(passToPacman(cmdArgs), 0)
+	err = config.Runtime.CmdRunner.Show(passToPacman(cmdArgs))
 	if err != nil {
-		return fmt.Errorf("%s %s", stderr, err)
+		return fmt.Errorf(gotext.Get("error updating package install reason to explicitly"))
```
Comment (Jguer) [inline install.go]: error updating package install reason to explicit

File: install.go (hunk 1 above)
Comment (Jguer) [reply install.go]: They still sound a bit weird but less that the existing

Comment (benjarobin) [reply install.go]: @Jguer Error message updated. Well, English is not my mother tongue... Thanks for the suggestions

**Comment by Jguer** [comment]
:+1: merging, thanks for your contribution

**Pull request merged**